pkg/api/config.go

```diff
@@ -75,13 +75,20 @@ func (config *ReleaseBuildConfiguration) ImageStreamFor(image string) (string, b
 }
 
 // DependencyParts returns the imageStream and tag name from a user-provided
-// reference to an image in the test namespace
-func (config *ReleaseBuildConfiguration) DependencyParts(dependency StepDependency) (string, string, bool) {
+// reference to an image in the test namespace. If a ClaimRelease is provided
```

Comment:
Ah, so you want to be able to call this "latest" but we need the _actual_ release name to be something unique so it does not collide in the namespace with the latest release that may be configured in the config for other job on the branch? Let's make the comment a bit more exhaustively clear please :)